_posts/2018-10-30-conducting-good-retrospectives.md

```diff
@@ -0,0 +1,110 @@
+
+#### Mid-Sprint Retrospective
+
+During the Mid-Sprint Retrospective, we engage in the [Lean Coffee](http://leancoffee.org) activity (unless someone suggests an alternate activity!). The chosen facilitator will explain the chosen activity and pass out the required supplies. 
+
+The basics of the Lean Coffee activity are:
+
+1. Generate discussion topics
+    - Limit to 5 minutes
+1. Vote on topics
+1. Discuss topics
+    - Limit to 5 minutes, then vote on continuing with a 2 minute extension, or else move on to next topic.
+    - Facilitator will take notes and track action items.
+
+Any topics that aren't discussed are carried over to the next meeting, unless the author states otherwise.
+
+#### Sprint Retrospective
+
+During the Sprint Retrospective we use the [Four Quadrant](http://www.funretrospectives.com/lessons-learned-quadrants-planning-vs-success/) activity. As with the Mid-Sprint retro, the chosen facilitator will explain the activity and pass out the required supplies.
+
+The basics of the Four Quadrant activity are:
+
+1. Enumerate events that happened during the last two weeks on sticky notes.
+    - Limit to 5 minutes
+1. The facilitator will draw a horizontal axis on the white board, with the left side indicating _bad_ and the right side _good_.
+1. Each person will give a brief description of their stickies and place them on the axis where they think they belong.
+1. The facilitator then draws a vertical axis, with the top indicating the event was _in the team's control_ and the bottom that it was _not in the team's control_.
+1. The facilitator quickly pulls each sticky to the appropriate place on the vertical axis. This is usually fairly obvious, but the rest of the team will signal where they think it should go in case of disagreement.
+
+At this point, the board should resemble something like this:
+
+![Diagram of a completed Four Quadrants diagram]({{ 'https://res.cloudinary.com/mhanberg/image/upload/v1541686229/four-quadrants.png' }})
+
+Starting in the top right corner and moving counter clockwise, the team discusses the events in an attempt to learn. If your team is prone to drawing out discussions, consider using a Lean Coffee style discussion timer.
+
+**Keep doing** - good and in your control: give yourselves a quick pat on the back to celebrate good work.
+
+**Improve** - bad and in your control: these are events that your team has the power to avoid in the future. The facilitator will lead discussion on the topics, driving toward actions that can be taken in the future.
+
+**Escalate** - bad and out of your control: these events can only be addressed by someone external to your team. Now is the time to figure out with who to escalate the issue, or assign someone the action of figuring that out later.
+
+**Kudos** - good and out of your control: these are positive events that were caused by someone external to your team. Here the team will assign an action to someone to reach out and express the team's gratitude.
+
```

Comment:
I feel like a sticky being "in the middle" or "on the line" is common enough that it deserves a mention and explanation. What do you think?